test-apps/presentation-test-app/src/frontend/index.tsx

```diff
@@ -32,13 +32,6 @@ export class SampleApp {
       // __PUBLISH_EXTRACT_START__ Presentation.Frontend.RpcInterface_2
       await ElectronApp.startup({ iModelApp: iModelAppOpts });
       // __PUBLISH_EXTRACT_END__
-    } else if (ProcessDetector.isBrowserProcess) {
-      await WebViewerApp.startup({
```

Comment:
the test app doesn't work.. don't we need to call `startup` on `IModelApp` or one of its specializations?
![image](https://user-images.githubusercontent.com/35135765/134294782-50f0ebff-b8e2-409b-92cc-8ef8d0ca6b14.png)

Reply:
Should be fixed now with latest push.